content/docs/handling-events.md

```diff
@@ -91,64 +90,66 @@ ReactDOM.render(
-  // Warning: this is *experimental* syntax.
+  // Այս կցումը պարտադիր է որպեսզի `this`-ը աշխատի հետկանչում
+  // Ուշադրություն\` սա *փորձարարական* շարահյուսություն է
   handleClick = () => {
-    console.log('this is:', this);
+    console.log('սա this-ն է`', this);
   }
 
   render() {
     return (
       <button onClick={this.handleClick}>
-        Click me
+        Սեղմել այստեղ
       </button>
     );
   }
 }
 ```
 
-This syntax is enabled by default in [Create React App](https://github.com/facebookincubator/create-react-app).
+Այս շարահյուսությունը լռությամբ միացված է [Ստեղծել React ծրագիր](https://github.com/facebookincubator/create-react-app)<sub>`eng`</sub>-ում։
 
-If you aren't using class fields syntax, you can use an [arrow function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in the callback:
+Եթե դուք չեք օգտագործում կլասի հատկության շարահյուսությունը, դուք կարող եք օգտագործել [սլաք ֆունկցիա](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions)<sub>`eng`</sub> ետկանչում\`
 
 ```js{7-9}
 class LoggingButton extends React.Component {
   handleClick() {
-    console.log('this is:', this);
+    console.log('սա this-ն է`', this);
   }
 
   render() {
-    // This syntax ensures `this` is bound within handleClick
+    // Այս շարահյուսությունը ապահովում է `this`-ի կցված լինելը handleClick-ին
     return (
       <button onClick={(e) => this.handleClick(e)}>
-        Click me
+        Սեղմել այստեղ
       </button>
     );
   }
 }
 ```
 
-The problem with this syntax is that a different callback is created each time the `LoggingButton` renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.
+Շարահյուսության խնդիրը կայանում է նրանում, որ ամեն անգամ, երբ `LoggingButton` արտապատկերվում է, մեկ այլ ետկանչ է ստեղծվում։ Մեծամասամբ դա լավ է։ Այնուամենայնիվ, եթե ետկանչը փոխանցված է որպես հենակ ավելի ցածր կոմպոնենտների համար, այդ կոմպոնենտները կարող են հանգեցնել հավելյալ կրկնակի արտապատկերման։ Ընդհանուր առմամբ մենք խորհուրդ ենք տալիս կցումը կատարել constructor-ում կամ օգտագործել հրապարակային կլասի հատկության շարահյուսությունը, որպեսզի խուսափեք նման արտադրողականության խնդիրներից։
 
-## Passing Arguments to Event Handlers {#passing-arguments-to-event-handlers}
+## Փոխանցել արգումենտներ իրադարձությունների մշակողին {#passing-arguments-to-event-handlers}
 
-Inside a loop it is common to want to pass an extra parameter to an event handler. For example, if `id` is the row ID, either of the following would work:
+Ցիկլի ներսում սովորաբար անհրաժեշտ է լինում իրադարձությունների մշակողին փոխանցել հավելյալ պարամետրներ։ Օրինակ, եթե `id`-ն տողի հերթական համարն է, ապա և՛ առաջին տարբերակը կաշխատի, և՛ երկրորդը\`
```

Comment:
Ի՞նչ «հերթական համար»

For example, if `id` is the row ID, either of the following would work -> Օրինակ, եթե `id`-ն տողի տարբերակիչն է, ապա ներկայացվածներից ցանկացածը կաշխատի։

Reply:
done